src/main/java/reposense/system/LogsManager.java

```diff
@@ -115,6 +122,14 @@ private static ConsoleHandler createConsoleHandler() {
         return consoleHandler;
     }
 
+    /** Set the {@code logFolderLocation } and add fileHandler
+     *  to all the {@code Logger} in {@code LOGGER_LIST}
```

Comment:
All the `{@code}` used here are all `private`, not visible outside of this class though.
Try to make the Javadoc not have any specific reference to any of those private variables.